go.mod

```diff
@@ -14,5 +14,6 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.0.0
 	go.uber.org/multierr v1.5.0
+	gopkg.in/yaml.v2 v2.3.0
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
```

Comment:
Do we need two versions?

Reply:
No we don't. The VS Code extension keeps inserting yaml.v2 for me...

Reply:
Weird. Does `go mod tidy` clean it up?

Reply:
Unfortunately, `go mod tidy` doesn't always clean up well.

Reply:
Goodbye to yaml.v2